assets/js/babyYoda.js

```diff
@@ -0,0 +1,47 @@
+
+const spawnBabyYoda = (randPosX, randPosY, colour) => {
```

Comment:
I think that with Baby Yoda, we always want him in the same position starting, so it might not be necessary to add the parameters here. I also mentioned the colour lower down - that shouldn't be required anymore since we now have the sprites.